src/l2_packet/l2_packet_zephyr.c

```diff
@@ -94,10 +94,9 @@ static void l2_packet_receive(int sock, void *eloop_ctx, void *sock_ctx)
 	// FIXME: sll_addr is not being filled as L2 header is not removed
 	hdr = (const struct ieee802_1x_hdr *) buf;
 
-	// FIXME: L2 header is now removed but sll_protocol is not set.
-	// So, as a workaround add this check to drop packets.
```

Comment:
AFAIK, this is still applicable, no?

Reply:
will add it back

Reply:
Sorry, didn't check the fix, looks like now L2 sll_protocol is set? If yes, you can remove the FIXME

Reply:
yes, updated, thanks